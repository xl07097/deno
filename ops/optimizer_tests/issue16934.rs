async fn send_stdin(
  state: &mut OpState,
  cmd: String,
) -> Result<(), anyhow::Error> {
  // https://github.com/denoland/deno/issues/16934
  //
  // OpState borrowed across await point is not allowed, as it will likely panic at runtime.
  let instance = state.borrow::<MinecraftInstance>().clone();
  instance.send_command(&cmd, CausedBy::Unknown).await?;
  Ok(())
}
