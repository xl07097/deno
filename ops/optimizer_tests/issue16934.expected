=== Optimizer Dump ===
returns_result: false
has_ref_opstate: false
has_rc_opstate: false
has_fast_callback_option: false
needs_fast_callback_option: false
fast_result: None
fast_parameters: []
transforms: {}
is_async: false
fast_compatible: false
